assets/src/edit-story/components/panels/layer/layerContent.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+const LayerBackground = styled.span`
+	opacity: .5;
+`;
+
+const LayerText = styled.span`
+	overflow: hidden;
+	white-space: nowrap;
+	text-overflow: ellipsis;
+`;
+
+function LayerContent( { element } ) {
+	switch ( element.type ) {
+		case 'text':
+			return <LayerText dangerouslySetInnerHTML={ { __html: element.content } } />;
+
+		case 'image':
+			// Disable reason: Well, it's actually an image element, so it's the best description
+			// eslint-disable-next-line jsx-a11y/img-redundant-alt
+			return <img src={ element.src } alt="Image element" height="20" />;
+
+		case 'video':
+			return <img src={ element.poster } alt="Video element" height="20" />;
+
+		case 'square':
+			return 'Square';
+
+		default:
+			return (
+				<LayerBackground>
```

Comment:
Not sure I understand this completely. A `type=image` or a `type=video` (for now) can be used as a background element. So, why is it used separate from other types? Wouldn't an image (for instance) already be handled above with `case 'image':` and never get here? I see below that this will work as you intend when the the `backgroundElement` is not yet defined. But what when it's already defined?

Reply:
In terms of the layer panel, the background layer is special regardless of it being empty, an image or a video. It always has the same description and icon.

Reply:
Fixed with other stuff in 07f0e2a.